### DISCUSSION TITLE
ulpoad error

### DISCUSSION THREAD

**Discussion by anshuld1995** [community/community#27562] [Other Feature Feedback, Questions, & Ideas]
hello everyone  , i want to know about uploading system in github , its provide only less then 25mb, so we cannot upload more than it ,

**Comment by jfullstackdev** [comment]
maybe these articles will help 

https://towardsdatascience.com/uploading-large-files-to-github-dbef518fa1a

https://docs.github.com/en/repositories/working-with-files/managing-large-files/about-large-files-on-github

**Reply by anshuld1995** [reply]
in the above article help alot thanku

**Reply by Gilbert1817** [reply]
Ok sir, guide me sir ba
